src/entity/task/new_task.rs

```diff
@@ -184,6 +184,39 @@ impl NewTask {
         }
     }
 
+    fn validate_vdaf_is_supported(
+        &self,
+        leader: &Aggregator,
+        helper: &Aggregator,
+        errors: &mut ValidationErrors,
+    ) {
+        if let Some(vdaf) = self.vdaf.as_ref() {
+            let name = vdaf.name();
+            if !leader.vdafs.contains(&name) || !helper.vdafs.contains(&name) {
+                match errors.errors_mut().entry("vdaf").or_insert_with(|| {
+                    ValidationErrorsKind::Struct(Box::new(ValidationErrors::new()))
+                }) {
+                    ValidationErrorsKind::Struct(errors) => {
```

Comment:
this awkwardness is due to the validate crate, and I'm increasingly itching to just drop it